CesiumGltf/include/CesiumGltf/PropertyTableView.h

```diff
@@ -107,21 +108,21 @@ class PropertyTableView {
    * @return A {@link PropertyTablePropertyView} of the property. If no valid property is
    * found, the property view will be invalid.
    */
-  template <typename T>
-  PropertyTablePropertyView<T>
+  template <typename T, bool Normalized = false>
```

Comment:
Yep, it does return an invalid `PropertyTablePropertyView`! But I'll update the documentation so that's more apparent. I also added a more proper `ErrorNormalizationMismatch` error code, separate from the `ErrorInvalidNormalization` code. (The latter should be used for floating point components erroneously marked as normalized ).